### ISSUE TITLE
regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by kumarharsh** [electronjs/windows-installer#8]
Just including the installer via `const electronInstaller = require('electron-winstaller');` throws this error:

```
ReferenceError: regeneratorRuntime is not defined
    at D:\code\project\node_modules\electron-winstaller\lib\index.js:9:31
```

**Comment by TemaSM** [comment]
Same with me, but I resolved it using:
`npm i babel-polyfill --save-dev`
`require('babel-polyfill')` (place it upper `require('electron-winstaller')`)
But after that, you will see another issue, at this time its: `Warning: electronInstaller.build is not a function`

**Comment by kumarharsh** [comment]
yes, because there is no `build` anywhere in the source code. :( I think the docs are misleading, it should be `electronInstaller.createWindowsInstaller`

**Comment by kumarharsh** [comment]
Although now looking at the electron-winstaller package.json, the babel-polyfill is there, and it uses the babel-maybefill which in turn uses babel-polyfill in the `src/` folder. I think the `lib/` folder is not updated to be built from the latest `src/` folder

**Comment by TemaSM** [comment]
I think it's compiled from `src/` folder, because if we look into `package.json`, may see:
[#L12](https://github.com/electronjs/windows-installer/blob/master/package.json#L12)
```
"scripts": {
    "compile": "babel -d lib/ src/",
```

**Comment by TemaSM** [comment]
Additionally here is a problem with code at `tasks` folder.

**Comment by paulcbetts** [comment]
Ugh, I'll fix this - this is a Babel bug we have to work around: https://phabricator.babeljs.io/T7128

**Comment by kevinsawicki** [comment]
Fix published in `2.0.2`
